docs/development/test-vectors.rst

```diff
@@ -146,6 +146,9 @@ Custom X.509 Request Vectors
   SHA256 generated using OpenSSL.
 * ``ec_sha256.pem`` - Contains a certificate request using EC (``secp384r1``)
   and SHA256 generated using OpenSSL.
+* ``san_rsa_sha256.pem`` and ``san_rsa_sha256.der`` - Contain a certificate
```

Comment:
Oops, updated PR to name the files properly and put the right information in the docs.